turbopack/crates/turbopack-ecmascript/src/references/esm/export.rs

```diff
@@ -139,6 +141,23 @@ pub async fn follow_reexports(
     let mut module = module;
     let mut export_name = export_name;
     loop {
+        // Do not go into `internal` fragments. Those modules are private to a single module.
+        if let Some(fragment) =
+            Vc::try_resolve_downcast_type::<EcmascriptModulePartAsset>(module).await?
+        {
+            let part = fragment.await?.part.await?;
+
+            if let ModulePart::Export(export) = *part {
+                if *export.await? == export_name {
+                    return Ok(FollowExportsResult::cell(FollowExportsResult {
+                        module,
+                        export_name: Some(export_name),
+                        ty: FoundExportType::Found,
+                    }));
+                }
+            }
+        }
+
```

Comment:
Shouldn't be needed?

Reply:
Currently, I am not sure about these changes. IIRC some tests failed when I removed this logic, but situation may be different.